frontend/src/metabase/common/components/EntityPicker/components/EntityPickerModal/EntityPickerModal.tsx

```diff
@@ -42,7 +42,7 @@ export interface EntityPickerModalProps<Model extends string, Item> {
   onItemSelect: (item: Item) => void;
   canSelectItem: boolean;
   onClose: () => void;
-  tabs: [EntityTab<Model>, ...EntityTab<Model>[]]; // Enforces that the array is not empty
+  tabs: EntityTab<Model>[]; // Enforces that the array is not empty
```

Comment:
probably want to remove the comment here

Reply:
26ad7d9b789eb3257971412273a0e96855e4276f